libs/vertexai/langchain_google_vertexai/vectorstores/document_storage.py

```diff
@@ -234,6 +234,7 @@ def __init__(
         kind: str = "document_id",
         text_property_name: str = "text",
         metadata_property_name: str = "metadata",
+        exclude_from_indexes: List[str] = [],
```

Comment:
please, try to avoid default mutable values. Let's change it to Optional[List[str]], please, and check for None.